Sources/MapboxMaps/Foundation/Extensions/Core/ResourceOptions.swift

```diff
@@ -88,41 +83,17 @@ public struct ResourceOptions {
     ///     exists in the cache.
     public init(accessToken: String,
                 baseURL: URL? = nil,
-                cachePathURL: URL? = nil,
+                dataPathURL: URL? = nil,
                 assetPathURL: URL? = nil,
-                cacheSize: UInt64 = Self.defaultCacheSize,
                 tileStore: TileStore? = nil,
                 tileStoreUsageMode: TileStoreUsageMode = .readOnly) {
         self.accessToken        = accessToken
         self.baseURL            = baseURL
-        self.cachePathURL       = cachePathURL ?? ResourceOptions.cacheURLIncludingSubdirectory()
+        self.dataPathURL        = dataPathURL
```

Comment:
so the default is changed to nil? In Android we set the default path to /.mapbox/maps/ cc @julianrex

Reply:
I think the default gl-native will use is `/.mapbox/`

Reply:
We are going to pass null dataPath to core by default, creating an Android PR at https://github.com/mapbox/mapbox-maps-android/pull/403